src/transformers/file_utils.py

```diff
@@ -1859,6 +1859,20 @@ def to_py_obj(obj):
         return obj
 
 
+def to_numpy(obj):
+    """
+    Convert a TensorFlow tensor, PyTorch tensor, Numpy array or python list to a Numpy array.
+    """
```

Comment:
maybe add 

```
    if isinstance(obj, (dict, UserDict)):
        return {k: to_py_obj(v) for k, v in obj.items()}
``` 

as well?

And could you maybe also add a:

```
elif is_flax_available() and _is_jax(obj):
    return np.asarray(obj)
```

Reply:
we should actually add this to `to_py_obj(obj)` as well

Reply:
Added :+1: